privaterelay/templates/profile.html

```diff
@@ -183,7 +183,7 @@
 
               <!-- delete email address -->
               <div class="column-delete">
-                <form action="{% url 'emails-index' %}" method="POST" class="delete-email-form xflx jst-cntr al-cntr" data-delete-relay="{{ relay_address.address }}@{{ RELAY_DOMAIN }}">
+                <form action="{% url 'emails-index' %}" method="POST" class="delete-email-form xflx jst-cntr al-cntr" data-delete-relay="{{ relay_address.address }}@{{ relay_address.domain_value }}">
```

Comment:
Yeah, I like the idea to use `relay_address.full_address` everywhere instead of using `{{ relay_address.address }}@{{ relay_address.domain_value }}`

Reply:
Done at https://github.com/mozilla/fx-private-relay/pull/1121/commits/f0d1093363f7cc27a7edc039be40f1ce9624f884